test/sanity/issue5980-aws-sdk-embedded-youtobe-video-crash/test.py

```diff
@@ -0,0 +1,48 @@
+import time
+import os
+import shutil
+import subprocess
+import platform
+import sys
+import shutil
+from subprocess import Popen, PIPE
+
+if platform.system() != 'Linux':
+    print 'Skipped for non Linux platform'
+    sys.exit(0)
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from nw_util import *
+
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+
+chrome_options = Options()
+testdir = os.path.dirname(os.path.abspath(__file__))
+chrome_options.add_argument("nwapp=" + testdir)
+node_module = os.path.join(testdir, "node_modules")
+os.chdir(testdir)
+
+try:
+    shutil.rmtree(node_module)
+except:
+    pass
+
+assert(False == os.path.isdir(node_module))
+subprocess.call(["npm", "install"])
```

Comment:
may need to use 'install_native_modules' in nw_utils here.

Reply:
updated